python/hail/dataset.py

```diff
@@ -750,6 +750,63 @@ def annotate_variants_expr(self, condition):
         pargs = ['annotatevariants', 'expr', '-c', condition]
         return self.hc.run_command(self, pargs)
 
+    def annotate_variants_keytable(self, keytable, condition, vdsKey=None):
```

Comment:
Ah, I see.  Yes, we definitely want the flexibility of expressions.  I was thinking we would have a list of expressions rather than a single expression which denoted a struct.  For this purpose, they wouldn't even need to be named.